keyboards/bpiphany/pegasushoof/2015/info.json

```diff
@@ -1,5 +1,11 @@
 {
-    "keyboard_name": "Majestouch TKL \\\\w The Pegasus Hoof 2015",
+    "keyboard_name": "Majestouch TKL The Pegasus Hoof 2015",
+    "manufacturer": "Filco",
+    "usb": {
+        "vid": "0x4245",
+        "pid": "0x6050",
+        "device_version": "1.0.0"
+    },
```

Comment:
If this isn't any different from what's in `keyboards/bpiphany/pegasushoof/info.json`, no need for it to be in this file.
```suggestion
```